app/src/org/commcare/adapters/MenuAdapter.java

```diff
@@ -270,9 +270,10 @@ public void setupTextView(TextView textView, MenuDisplayable menuDisplayable) {
     }
 
     public void setupImageView(ImageView mIconView, MenuDisplayable menuDisplayable) {
-        String imageURI = menuDisplayable.getImageURI();
-        Bitmap image = MediaUtil.inflateDisplayImage(context, imageURI);
         if (mIconView != null) {
+            int iconDimension = (int)context.getResources().getDimension(R.dimen.menu_icon_size);
+            Bitmap image = MediaUtil.inflateDisplayImage(context, menuDisplayable.getImageURI(),
+                    iconDimension, iconDimension);
```

Comment:
Not actually using the new flag here because `mIconView` imposes the precise dimensions anyway, but realized that the MenuAdapter code should at least be using `iconDimension` as a rough bounding dimension